### PR TITLE
Added Github MarketPlace Cards Box-shadow

### PR DESCRIPTION
**Pull request by HooriaHIC**

### CODE REVIEW

**Comment by HooriaHIC** [comment]
#175

**Comment by HooriaHIC** [comment]
#45

File: src/shadows.js [hunk 1]
```diff
@@ -433,6 +433,11 @@ const shadows = [
     name: "Starbucks",
     shadow: "0 11px 15px 0 rgba(0, 0, 0, 0.1), 0 9px 46px 8px rgba(0, 0, 0, 0.06), 0 24px 38px 3px rgba(0, 0, 0, 0.07)",
     border: "1.5px solid rgba(0, 0, 0, .12)"
+  },
+  {
+    "name": "Github MarketPlace Card Box-shadow",
+    "shadow": "0 4px 10px 1px rgba(0,0,0,.2)",
+    "border": "0"
   }
```
Comment (nakulrathore) [inline src/shadows.js:441]: add a comma ending last item of the array.

**Review by nakulrathore — CHANGES_REQUESTED** [review]
update the file with requested comment.

**Comment by HooriaHIC** [comment]
I've Submited another pull request with the changes you requested :)

**Pull request closed without merging**